### PR TITLE
Added page witn Epsilon in Ecore details

### PR DESCRIPTION
**Pull request by arcanefoam**
This adds the documentation for Ecore Delegates

### CODE REVIEW

File: mkdocs/docs/doc/articles/index.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ This page links to more than **60 articles** on various aspects of Epsilon. Shou
 - [Parsing XML documents as EMF models with Flexmi](../flexmi): This article demonstrates how you can use Flexmi to parse XML documents in a fuzzy manner as instances of Ecore metamodels.
 - [Modularity Mechanisms in Flexmi](modular-flexmi): This article demonstrates how you can break down Flexmi models over multiple files and use templates to capture complex reusable structures in your models.
 - [Connecting to CDO repositories](cdo-emc.md): This article shows how to connect to an Eclipse Connected Data Objects model repository from Epsilon.
+- [Using Epsilon for ECore Validation, Setting and Invocation Delegation](epsilon-in-ecore): This shows how to use Epsilon in an ECore model to provide Invocation and Setting delegation with EOL and Validation DElegation with EVL.
```
Comment (kolovos) [inline mkdocs/docs/doc/articles/index.md:47]: ECore -> Ecore (2x)
DElegation -> Delegation

File: mkdocs/docs/doc/articles/epsilon-in-ecore/index.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
+# Using Epsilon for ECore Validation, Setting and Invocation Delegation
+
+EMF provides three delegation mechanisms that enable functionality not directly supported by EMF to be delegated to a technology that can support it.
```
Comment (kolovos) [inline mkdocs/docs/doc/articles/epsilon-in-ecore/index.md:3]: Add a note that this feature is available in versions >= 2.5.0?

File: mkdocs/docs/doc/articles/epsilon-in-ecore/index.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
+# Using Epsilon for ECore Validation, Setting and Invocation Delegation
```
Comment (kolovos) [inline mkdocs/docs/doc/articles/epsilon-in-ecore/index.md:1]: ECore -> Ecore

Comment (arcanefoam) [reply mkdocs/docs/doc/articles/epsilon-in-ecore/index.md]: Fixed

**Pull request merged**